nidata/multimodal/hcp/hcp.rst

```diff
@@ -1 +1,19 @@
-TODO: HCP needs a RST file.
\ No newline at end of file
+HCP 900 Subjects Release Data
+
+
+Notes
+-----
+The Human Connectome Project seeks to map out the region-to-region structural and functional connections in the human brain. The 900 Subjects Release Data includes imaging data from over 900 healthy adult subjects collected from 2012 to spring of 2015. The data set not only includes MR scans, but also over 95 subjects with resting-state and task MEG scans.
```

Comment:
I suggest reading:
https://humanconnectome.org/about/project/

I also did a Google search, clicked on "News". There were lots of lay articles on the project. For example:
http://www.medicaldaily.com/10-faq-about-human-connectome-project-astonishing-sister-nihs-human-genome-project-362650

Please read and use the knowledge to give a better sense of that the project is, why it was done, a few results that came out of it, and what data we can make available here. For example, there are twin and genetic data in there, as well as very interesting background data that can be used to associate with the brain measures.